src/heliaServer.ts

```diff
@@ -33,7 +33,7 @@ export class HeliaServer {
   /**
    * Initialize the HeliaServer instance
    */
-  async init (): Promise<void> {
+  init = async (): Promise<void> => {
```

Comment:
can remove all these changes, was just testing if some of the `this` scope was getting lost and causing some errors.

Reply:
removed.